src/finance_quote/base.py

```diff
@@ -1,42 +1,41 @@
 import datetime
+import logging
 
 import requests
 
 
 class FinanceQuoteError(Exception):
     pass
 
+
 class SymbolNotFoundError(FinanceQuoteError):
     pass
 
+
 class SourceConnectionError(FinanceQuoteError):
     pass
 
+
 class Symbol:
     """Class to represent a symbol (currency, stock, index, ...)"""
     pass
 
-class Quote:
```

Comment:
ok, so this can be moved to class based field for documentation.
Having a self.value default to 0 is dangerous as if someone forgets to set it, it will have some value ==> better to have None (even better to not have anything to it raises an AttributeError but this is no option for you, correct?)
why is self.datetime a datetime and not a date ? if it is a datetime, is it a naive datetime or a lozalized datetime ? is it the time of publication ? the time of scraping ? if it is a closing price, what would be the value of the time part ?
what is f::q doing in this respect ?
for indices which have no currency, currency should be None ? if so, you need to fix the __repr__ for this case

Reply:
I let you in your branch apply the changes you already did + add documentation in the class answering the questions here above, OK?